pylearn2/devtools/tests/test_format.py

```diff
@@ -78,6 +78,14 @@
     "sandbox/lisa_rl/bandit/classifier_bandit.py",
     "sandbox/lisa_rl/bandit/classifier_agent.py",
     "sandbox/lisa_rl/bandit/plot_reward.py",
```

Comment:
Actually, I take back what I said earlier. Since this is sandbox and they're new files I'm OK with it.